Makefile

```diff
@@ -311,7 +311,7 @@ tftpd_start:
 	sudo true
 	@if sudo which atftpd >/dev/null ; then \
 		echo "Starting aftpd"; \
-		sudo atftpd --verbose --bind-address $(TFTP_IPRANGE).100 --daemon --logfile /dev/stdout --no-fork --user $(shell whoami) $(TFTPD_DIR) & \
+		sudo atftpd --verbose --bind-address $(TFTP_IPRANGE).100 --daemon --logfile /dev/stdout --no-fork --user $(shell whoami) --group $(shell whoami) $(TFTPD_DIR) & \
```

Comment:
This change breaks on any computer which doesn't have a group the same as a person's username (which is why it was removed :-P). Is there a way to get a users primary group?

Reply:
I think we want 

```shell
--group $(shell id -gn)
```